flake.nix

```diff
@@ -16,7 +16,9 @@
     cardano-node-exe = {
       url = "github:input-output-hk/cardano-node/ea8b632820db5546b22430bbb5ed8db4a2fef7dd";
     };
+    # Repository with network parameters
     cardano-configurations = {
+      # Override with "path:/path/to/cardano-configurations";
```

Comment:
I can't put this comment where it should really go as the lines aren't part of the changes yet, but the easiest way to override `cardano-configurations` is to just add it to `defaultConfig` below:

```
defaultConfig = final: with final; {
  inherit (inputs) cardano-configurations;
  network = "testnet";
  node = { port = 3001; };
  # ...
};

```